src/utils/source-maps.js

```diff
@@ -2,7 +2,22 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
 
+//@flow
 import { getSource } from "../selectors";
+import type { SourceRecord } from "../reducers/sources";
+import type { Location } from "../types";
+
+export function getGeneratedSourceId(originalId: string) {
+  const match = originalId.match(/(.*)\/originalSource/);
+  return match ? match[1] : "";
+}
+
+export function isOriginal(source: SourceRecord) {
+  if (!source) {
+    return false;
+  }
+  return !!getGeneratedSourceId(source.id);
+}
 
```

Comment:
Might not need these extra utils

Reply:
isOriginal and getGeneratedSourceId are used in _utils/editor/index.js_ in `shouldShowFooter`

Reply:
There already exists `isOriginalId` util which can solve the issue in `shouldShowFooter` ([see the comment](https://github.com/devtools-html/debugger.html/pull/4636/files#r150379991)). These two utils feel like alot for what we need.

Reply:
true